src/actions/AuthActions.js

```diff
@@ -13,9 +13,9 @@ import {
 } from './types';
 
 import {FACEBOOK_AUTH} from '../../env';
-import {tokens} from '../constants';
+import {tokens, navKeys} from '../constants';
```

Comment:
Since the `AuthActions` will be navigating us around, they need access to the `navKeys`